packages/opentelemetry-semantic-conventions-ai/opentelemetry/semconv_ai/_testing.py

```
@@ -439,3 +439,27 @@ def test_watsonx_metrics_kept(self):
+    def test_aipou_work_receipt_id(self):
+        assert SpanAttributes.AIPOU_WORK_RECEIPT_ID == "aipou.work_receipt_id"
+
+    def test_aipou_evidence_class(self):
+        assert SpanAttributes.AIPOU_EVIDENCE_CLASS == "aipou.evidence_class"
+
+    def test_aipou_scheme(self):
+        assert SpanAttributes.AIPOU_SCHEME == "aipou.scheme"
+
+    def test_aipou_validation_status(self):
+        assert SpanAttributes.AIPOU_VALIDATION_STATUS == "aipou.validation_status"
+    
```

Ruff (0.15.21)

[warning] No newline at end of file

Add trailing newline

(W292)